rust/ql/src/queries/summary/SummaryStatsLess.ql

```diff
@@ -0,0 +1,18 @@
+/**
+ * @name Summary Statistics Less
+ * @description A table of summary statistics about a database, with data that
+ *              has been found to be noisy on tests removed.
+ * @kind metric
+ * @id rust/summary/summary-statistics-less
```

Comment:
`SummeryStatsReduced.ql` and `ust/summary/summary-statistics-reduced` instead?

Reply:
Yes I think I like that a bit more.  Updated.